test/road_test.py

```diff
@@ -65,3 +65,37 @@ def test_transition_probs_with_invisible_obstacle(obst, action):
     assert sum_probs == pytest.approx(1.0)
     assert probs[0] == max(probs)
     assert probs[1:] == [obst.prob_of_appearing() / 4] * 4
+
+
```

Comment:
Add speed as a parameter and test with multiple speeds.

Reply:
Done! Test passed 👍